src/main.tsx

```diff
@@ -1,10 +1,92 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
 import App from "./App.tsx";
-// import './index.css'
+import { ThemeProvider, createTheme } from "@mui/material";
+
+import "@fontsource/chakra-petch";
+
+const theme = createTheme({
+  palette: {
```

Comment:
try to add custom `primary {}` color and maybe some others (this should update text filelds color)

Reply:
✅